examples/compare_all_ids.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# Parses License Texts passed into them and compare them against all the License ids. 
+
+import os
+import sys
+import argparse
```

Comment:
Remove unnecessary imports from all the files.
(There are many libraries out there which does this automatically)

Reply:
Removed the sys import which was unnecessary